src/main/java/bio/terra/validation/BillingProfileRequestValidator.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.validation;
+
+import bio.terra.model.BillingProfileRequestModel;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import javax.validation.constraints.NotNull;
+import java.util.regex.Pattern;
+
+@Component
+public class BillingProfileRequestValidator implements Validator {
+
+    private static final String VALID_BILLING_ACCOUNT_ID_REGEX = "[A-Z0-9]{6}-[A-Z0-9]{6}-[A-Z0-9]{6}";
```

Comment:
This is specific to Google, right? Maybe comment on that so if we search later we know to fix it.

Reply:
Will do